editor/editor_settings.cpp

```diff
@@ -529,7 +529,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	set("text_editor/highlighting/highlight_all_occurrences", true);
 	set("text_editor/cursor/scroll_past_end_of_file", false);
 
-	set("text_editor/indent/type", 0);
+	set("text_editor/indent/type", "Tabs");
```

Comment:
Yeah that sounds like a better idea, then we can remove the inline ifs when retrieving the setting, as I doubt we will be adding any more types, unless someone wants semicolon based indentation.